src/verifierMiddleware.ts

```diff
@@ -1,4 +1,4 @@
-import express from 'express';
+import * as express from 'express';
```

Comment:
Can we change this to `import express from 'express'` ?

we can add this `"esModuleInterop": true,` to our current `tsconfig.json` under `compilerOptions`.